### DISCUSSION TITLE
private/clrs

### DISCUSSION THREAD

**Discussion by giscus** [imarvinle/CSGuide#41] [Announcements]
# private/clrs

算法导论 (第四版) PDF 高清中文版下载，算法导论 PDF，CLRS PDF 下载

https://csguide.cn/private/clrs.html

**Comment by Po00iu6** [comment]
算法导论建议NOIP后再看，因为这玩意其实就是补充知识的，里面大多数内容都学过了，而且如果上来就看着玩意相当于把人往坑里推。蒟蒻还是把底子打牢，充分利用好时间才能打好OI

**Reply by imarvinle** [reply]
👍
